roles/core/time/templates/chrony.conf.j2

```diff
@@ -43,12 +47,14 @@ rtcsync
 
 {% if time_profile == 'server' %}
 
-# Allow NTP client access from local network.
+# Allow NTP client access from local network.*
+{% if network_interfaces is defined %}
```

Comment:
I believe `network_interfaces` is a mandatory parameter. Can you confirm @oxedions ?